editor.planx.uk/src/components/Header.tsx

```diff
@@ -584,9 +584,9 @@ const Header: React.FC = () => {
   const pathnameArray = window.location.pathname.split("/");
 
   useEffect(() => {
-    const findPathname = pathnameArray.find(
-      (pathname) => pathname === "preview" || pathname === "draft",
-    );
+    const findPathname =
+      pathnameArray.pop() === "preview" || pathnameArray.pop() === "draft";
```

Comment:
`.pop()` is a mutating method - `pathname` won't remain consistent for both these checks.

![image](https://github.com/user-attachments/assets/5c25ced2-a5b8-4ad5-b261-23d0e7a8df7b)

![image](https://github.com/user-attachments/assets/f3cf8560-1c14-49c1-8557-bcc30d210a69)